### PR TITLE
Update finalizing settup page (and add troubleshooting section) to avoid unnecessary CTR transfers on section II

### PR DESCRIPTION
**Pull request by TheGoofBall64**
updates finalizing setup page section II to link to a (new) troubleshooting section to avoid unnecessary CTRtransfers when your device fails to update

adds troubleshooting section on the troubleshooting page.
changes finalizing setup page to link to this troubleshooting section.

### CODE REVIEW

**Comment by win9x-se** [comment]
a few things

Unplugging routers can cause a ton of problems, i.e. needing to reflash or otherwise fix it, not all routers are made to be unplugged on whim

Not everyone has access to their router/internet system and thus can't do things like disable router wide firewall, etc

The 3DS itself doesn't have a firewall, so that wouldn't apply

Most of these steps are redundant as a lot of the time, it comes down to "bring your console closer, and reboot".

**Comment by MechanicalDragon0687** [comment]
the more recent issues we have had with people updating is due to their wifi connection settings still being set for the dns hacks or being broken. I have had my connection fail even after clearing the dns settings and had to recreate it completely to fix it.

Perhaps adding a Wifi Troubleshooting section on the troubleshooting page would make more sense considering there are several methods that require wifi connection manipulation now?

**Comment by TheGoofBall64** [comment]
Yeah. Your probably right. I'll add something like that

**Comment by TheGoofBall64** [comment]
> Unplugging routers can cause a ton of problems, i.e. needing to reflash or otherwise fix it, not all routers are made to be unplugged on whim

Yeah your right. I'll fix that

> Not everyone has access to their router/internet system and thus can't do things like disable router wide firewall, etc

This is kind of a redundant argument. Yes it is almost never the problem, but If you are having problems and have access to, and the know how, to disable your firewall, go ahead and try. it shouldn't matter if some people can not access it. It's still a potential problem.

> The 3DS itself doesn't have a firewall, so that wouldn't apply

Yes, the 3ds doesn't have any sort of firewall. But your WiFi itself could have a firewall, and could block specific things your 3ds calls for. As per Nintendo points out in their own network troubleshooting guide

> Most of these steps are redundant as a lot of the time, it comes down to "bring your console closer, and reboot".

Yes. You are right. But again, I myself has had to restart my WiFi network multiple times. Even after going closer and rebooting. And even had to try another network entirely because my WiFi just didn't work that day. This is why (I'm going to) adding a troubleshooting section on there like Randal suggested would be a good idea. Most of the time it does come down to "bring your console closer, and reboot". But it still happens

**Comment by MechanicalDragon0687** [comment]
adding a "check the firewall settings in your router" is more likely to confuse the common user of the guide. If they have firewall settings in the router blocking things, and they set them up, it is likely unnecessary to tell them to check them. if they have firewall settings in the router and they didnt, telling them to check it is moot

**Comment by TheGoofBall64** [comment]
Ok. I went add ''check firewall" then

**Comment by TheGoofBall64** [comment]
If you have any other suggestions on what I should add to the troubleshooting section please go ahead. Thank you

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:161]: This is a statement. It shouldn't be a 'step' in the troubleshooting.

Comment (lilyuwuu) [reply _pages/en_US/troubleshooting.txt:161]: `Eshop` -> `eShop`; `ect.` -> `etc.`

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:163]: "Auto" should be capitalized

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:165]: Use consistent capitalization of WiFi

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:166]: `ect.` -> `etc.`

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:162]: Remove the comma

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:167]: This doesn't really work as an `etc.` If your only example is Nintendo servers being down, use `Nintendo servers may be down. Try again later` or something.

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:168]: "this" is ambiguous in this context; needs to be reworded

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:165]: `Sign in` implies that there's a portal/login page, which the 3DS doesn't support. Use `connect to` instead.

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:169]: The order of this steps should be from easiest to hardest to do, which would be (IMO):

1. Set DNS to Auto
2. Move closer to router
3. Delete and re-add WiFi network
4. Try a different WiFi connection
5. Nintendo's servers might be down
6. CTRTransfer
7. Ask for help

File: _pages/en_US/finalizing-setup.txt
Comment (lilyuwuu) [inline _pages/en_US/finalizing-setup.txt:70]: You are suggesting multiple ideas, each of which should be tried before updating again. This isn't necessarily clear from the text of your troubleshooting guide.

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:167]: `ect.` -> `etc.`

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:169]: I don't think `Try` should be at the beginning of every sentence (use the form used with CTRTransfer, which is `then try again`). It's left unspoken what to do after you reboot you WiFi router/set DNS to auto/whatever, which is unclear.

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:159]: This is a bit vague. The only case we're really worried about is system updating, as Nintendo doesn't offer any error codes when something goes wrong during system updates. If an error occurs in the browser or in eShop, there is usually an error code that goes along with it. We're not dealing with browser or eShop connection issues in this portion of the guide, so I don't think it should be so vague.

This also means an online services ban could be misinterpreted as a WiFi connection error.

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:165]: A user should only have one WiFi connection that they should be connecting to at this moment. I don't see the need to delete *multiple* connections.

**Review by lilyuwuu — CHANGES_REQUESTED** [review]

**Review by TheGoofBall64 — COMMENTED** [review]
fixed.

but I never put in to try and hold `A+R+L+Up(DPad) on boot (to attempt an update in recovery mode)` anywhere. and I'm having trouble on how to implement it. because the ideal situation is to try this on every option after "move closer to your WiFi". do you have any ideas on how to implement this

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:169]: The beginning of each point should be capitalized.

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:167]: "Connect" is misspelled, as is "connection". 

"Connect to a different WiFi connection" is a bit repetitive, but the 3DS calls them 'connections' so it *should* be fine...

It's a bit awkward to use `etc.` when you only have one item as an example. If the only 'different WiFi connection' you can think of is a mobile hotspot, it might be better to just word it as "Connect to a different WiFi connection, like a mobile hotspot [or something else]"

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:161]: An error when trying to use WiFi during system update, as opposed to trying to use something else?...

The best way to clarify this section would be to show that the error that people should see is "An error has occurred".

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:162]: option -> step; if it's an option, then people can follow them out of order (though they likely wouldn't)

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:165]: A bit awkwardly worded IMO. 'm partial to using the term `reconnect` but it might be unnecessarily vague.

**Review by lilyuwuu — CHANGES_REQUESTED** [review]

**Comment by lilyuwuu** [comment]
A Safe Mode step could just be
`1. Update from Safe Mode by turning off the console, holding L+R+Up+A on boot, and following the prompts` or something

File: _pages/en_US/troubleshooting.txt
Comment (TheGoofBall64) [reply _pages/en_US/troubleshooting.txt:169]: oh god. how did i miss that?

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:162]: Maybe replace these two bullet points with a paragraph that reads

"Occasionally, updates may fail to install after installing CFW. To fix this, reboot your 3DS after each step of this section, then try updating again."

This would be in line with other troubleshooting steps that add more information. 

Alternatively, since this is currently only linked on Finalizing Setup, you could just have " Reboot your 3DS after each step, then try updating again" as one sentence.

Third idea: Don't have any text before the steps, then add "then try updating again" after each step.

**Review by lilyuwuu — COMMENTED** [review]
The rest generally looks good to me!

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:161]: This shouldn't be a bullet point/indented.

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:169]: Is this intentional, as some way to discourage CTRTransfering? It's inconsistent to the rest of the text.

Also, "then try updating again" shouldn't be necessary because it's said at the beginning of this section.

Comment (TheGoofBall64) [reply _pages/en_US/troubleshooting.txt:169]: no it wasn't intentional. sorry, ill fix that

File: _pages/en_US/troubleshooting.txt
Comment (TheGoofBall64) [reply _pages/en_US/troubleshooting.txt]: fixed

**Review by lilyuwuu — APPROVED** [review]
Looks good

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:165]: A bit picky, but the guide uses the order of `(Left Shoulder) + (Right Shoulder) + (D-Pad Up) + (A)` to describe the buttons.

Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:165]: There should be a hyphen between on and screen (on-screen).

**Comment by TheGoofBall64** [comment]
when will this get merged? It's been 6 days since any significant changes were made. and it was approved

**Comment by lilyuwuu** [comment]
No one else has looked at it so I'm a bit hesitant to merge it. The policy is usually to get at least a couple people to look before merging. That's also why if you look at the PR history a lot of the changes that weren't to fix a typo or something smaller took a while to merge.

That being said, nothing will get done without a strict time limit so I'll go ahead and merge this in about 48 hours if no one else has any thoughts/opposition.

**Comment by win9x-se** [comment]
troubleshooting seems to have an error with the numbering
image

**Comment by MechanicalDragon0687** [comment]
> troubleshooting seems to have an error with the numbering
> image

this is normal. the numbering isnt hardcoded until built.

**Comment by win9x-se** [comment]
> 
> 
> > troubleshooting seems to have an error with the numbering
> > image
> 
> this is normal. the numbering isnt hardcoded until built.

yeah i just realized that, sorry

File: _pages/en_US/troubleshooting.txt
Comment (GlaZedBelmont) [inline _pages/en_US/troubleshooting.txt:161]: We usually use "console" or "device" when talking about the 3DS, the rest of your edits don't say 3DS

File: _pages/en_US/troubleshooting.txt
Comment (GlaZedBelmont) [inline _pages/en_US/troubleshooting.txt:169]: small nitpick but why do you say "[...] may be down. Try again later" then "[...], then try again" maybe using a comma on the former would be better

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [reply _pages/en_US/troubleshooting.txt:165]: That is incorrect -- in English, verb tenses should be consistent. The first instance is "turning" (off the console), so the next ones should indeed be "holding" and "following". "Update" is separate.

File: _pages/en_US/troubleshooting.txt
Comment (lilyuwuu) [reply _pages/en_US/troubleshooting.txt:169]: It'd technically be grammatically incorrect because "try again later" is a full sentence. You can use a semicolon to separate them though.

File: _pages/en_US/troubleshooting.txt
Comment (TheGoofBall64) [reply _pages/en_US/troubleshooting.txt:165]: I speak English as my main language. and even I never noticed this. I automatically write it with the correct verb tenses and everything

**Pull request merged**
